### DISCUSSION TITLE
Is it possible to retrieve info about the initial message from a reply message?

### DISCUSSION THREAD

**Discussion by jcordiez** [Secreto31126/whatsapp-api-js#386] [Q&A]
When receiving a reply, I see there is a context object containing what I suppose is the id of the original message, but that's pretty much all we have. Is there a way we can get info about the initial message's content, besides storing it ourselves somehow?

**Comment by Secreto31126** [comment]
Unfortunately, the API doesn't offer more information in the [`context` property of the webhook payload](https://whatsappapijs.web.app/types/types.ServerMessage.html), and there is no way to retrieve previous messages on the API as far as I know.

If I have to guess, I would say this is an intentional design choice from their side, as they aren't super keen on preserving messages for "privacy reasons", which makes quite useless the reply feature :/

**Reply by jcordiez** [reply]
I think your guess is right, makes sense. Thanks a lot for the quick and kind reply, and for the great work on this library !
Have a good day!
